crowbar_framework/app/assets/javascripts/jquery/nodeList.js

```diff
@@ -359,6 +397,35 @@
     this.dataBag.removedOld = true;
   };
 
+  NodeList.prototype.equalOrMatch = function(value, maybe_regexp) {
+    // First group contains the RegExp without pre and post '/'
+    var is_re = /^\/(.*)\/$/.exec(maybe_regexp);
+    if (is_re) {
+      var re = new RegExp(is_re[1]);
+      return re.test(value)
+    }
+
+    // Check if is an operator
+    var op_value = maybe_regexp.split(" ", 2);
+    if (op_value.length == 2) {
+      switch (op_value[0]) {
+        case "==":
```

Comment:
I would skip that. If there is no operator defined at the beginning i would always take the equal check. Just add a ```default``` section.